changes/1536.bugfix

```diff
@@ -0,0 +1,3 @@
+Add missing translations for validation:
```

Comment:
@RabiaSajjad could you also just note the "fix" to adding the validation fields to the Info schema? Thanks!